src/common/utils/compare.ts

```diff
@@ -106,25 +101,16 @@ export function getStateNonMetroCounties(
     .map(getLocationObj);
 }
 
-function getCountyObj(countyFips: string) {
-  const region = regions.findByFipsCode(countyFips);
-  assert(region, 'Missing region');
-  return [getLocationObj(region)];
-}
-
-function isNeighboringCounty(region: Region, countyFips: string) {
-  const neighboringCountiesFips = getAdjacentCounties(countyFips);
-  return neighboringCountiesFips.includes(region.fipsCode);
-}
-
 export function getNeighboringCounties(
   countyFips: string,
 ): SummaryForCompare[] {
-  const adjacentCounties = regions.counties
-    .filter((region: Region) => isNeighboringCounty(region, countyFips))
-    .map(getLocationObj);
-  const currentCounty = getCountyObj(countyFips);
-  return [...adjacentCounties, ...currentCounty];
+  const county = regions.findByFipsCodeStrict(countyFips) as County;
+  const adjacentCounties = county.adjacentCountyFips.map(fips => {
+    const region = regions.findByFipsCodeStrict(fips);
+    return getLocationObj(region);
+  });
```

Comment:
We can do this later, but here is an idea. The `County` objects have a `state` property with a `State` object, we could do the same for `adjacentCounties` to avoid having to look up by FIPS each time.